docs/release-notes/release-notes-0.18.5.md

```diff
@@ -37,6 +37,12 @@
 ## RPC Updates
 
 ## lncli Updates
+
+* [Add a new flag](https://github.com/lightningnetwork/lnd/pull/9491)
```

Comment:
Don't think we need to have this in a minor release unless it's an important bug fixes.

Reply:
changing to 19